lighting/models/product_source_line.py

```diff
@@ -20,6 +20,60 @@ class LightingProductSourceLine(models.Model):
     _rec_name = "type_id"
     _order = "sequence"
 
+    @api.model
```

Comment:
Maybe using property will be similar to a constant.
```suggestion
    @property
```
chack for examples in Odoo